deploy/bundle/manifests/ceph-conf.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  ceph.conf: |
+    [global]
+    auth_cluster_required = cephx
+    auth_service_required = cephx
+    auth_client_required = cephx
+  # keyring is a required key and its value should be empty
+  keyring: |
+metadata:
+  name: ocs-metrics-exporter-ceph-conf
```

Comment:
just checking, we dont need a namespace here, right?

Reply:
No, it should be created in a namespace where bundle is installed. Namespace is not allowed here.